src/frontend/components/UI/HowLongToBeat/index.tsx

```diff
@@ -0,0 +1,64 @@
+import { HowLongToBeatEntry } from 'howlongtobeat'
+import React, { useEffect } from 'react'
+import { useTranslation } from 'react-i18next'
+import './index.scss'
+
+// react component with 3 square with how long to beat info
+
+type Props = {
+  title: string
+}
+
+export default function HowLongToBeat({ title }: Props) {
+  const [howLongToBeatInfo, setHowLongToBeatInfo] =
+    React.useState<HowLongToBeatEntry | null>(null)
+  const { t } = useTranslation('gamepage')
+
+  useEffect(() => {
+    window.api.getHowLongToBeat(title).then((howLongToBeatInfo) => {
+      if (howLongToBeatInfo) {
+        setHowLongToBeatInfo(howLongToBeatInfo)
+      }
+    })
+  }, [title])
+
+  if (!howLongToBeatInfo) {
+    return null
+  }
+
+  const { gameplayMain, gameplayMainExtra, gameplayCompletionist } =
+    howLongToBeatInfo || {}
```

Comment:
Is this `|| {}` needed? we should already by a non-null/undefined/false values from the previous if